### DISCUSSION TITLE
Arco-VUE组件库的modal对话框如何函数调用显示自定义组件？

### DISCUSSION THREAD

**Discussion by kustone** [arco-design/arco-design-vue#746] [Q&A]
#### 官方示例是arco.$modal.info(title:str,content:str)的例子，没有展示如和渲染自定义组件
` 
  this.$modal.info({
        title: 'Info Title',
        content: 'This is an info message'
      });
`

### 以下是我的代码方法：

`			
	import newEle from '@/views/layout/component/new.vue'；
`
`
this.$modal.open({
    title: "测试标题",
   content:()=>{
   return newEle.render()
 }
})
`
#### 实际显示结果是基础的html标签（如div/p等）和css样式能正常显示，但是arco的组件标签就无法正常渲染（比如a-button等）

**Comment by flsion** [comment]
@kustone 你好，renderfunction需要返回Vnode类型，一般是 h 函数或者 JSX。这里可以写成 ()=> h(newEle,{...props})。
https://vuejs.org/api/render-function.html#h

**Comment by binzc2004** [comment]
我也遇到了真的恶心，用最基础的input能够正常显示，用arcodesign的组件就不能显示
